src/libtelio.udl

```diff
@@ -23,14 +23,14 @@ enum RttType {
 };
 
 [Error]
-enum TelioError {
-    "UnknownError",
-    "InvalidKey",
-    "BadConfig",
-    "LockError",
-    "InvalidString",
-    "AlreadyStarted",
-    "NotStarted"
+interface TelioError {
```

Comment:
Why is this an interface now?

Reply:
You can't expose the fields of the rust enum if you use a udl enum, you have to use a udl interface in that case

https://mozilla.github.io/uniffi-rs/udl/errors.html#throwing-errors